vcs/git/patch.go

```diff
@@ -8,6 +8,17 @@ import (
 
 // patch -p0 < diff.patch
 
+func isFileBinaryGit(filename string) bool {
```

Comment:
To throw that error:

`func isFileBinaryGit(filename string) (bool, error)l {`